config/peerpods/podvm/gcp-podvm-image-handler.sh

```diff
@@ -109,9 +108,7 @@ function create_image() {
 }
 
 function set_image_version_and_name() {
-  # Set the image version
-  # It should follow the Major(int).Minor(int).Patch(int)
-  IMAGE_VERSION="${IMAGE_VERSION_MAJ_MIN}.$(date +'%Y%m%d%S')"
+  IMAGE_VERSION="$(date +'%Y%m%d%S')"
```

Comment:
You can add a comment here based on what is expected by GCP.
Like `Major(int).Minor(int).Patch(int)` is expected by Azure